enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -168,14 +171,15 @@ $ curl -k -i -s -X POST -u admin:$PASSWORD https://$SUPERHUB/api/fr/setup-hub/su
 $ curl -k -i -s -X POST -u admin:$PASSWORD https://$FEEDER/api/fr/setup-hub/feeder
 ```
 
-### Enable feeder, without API
+### Enable feeder manually
```

Comment:
I think the original title makes more sense. *without API* or *only using policy* makes most sense to me.